### PR TITLE
Complete Contour Examples

### PR DESCRIPTION
**Pull request by caimichael**
I remember you mentioning that you might want contour7() refactored to more closely follow PlotlyJS.jl conventions, but I couldn't quite see how. Feedback on that one would be appreciated!

### CODE REVIEW

**Comment by spencerlyon2** [comment]
Hey @caimichael this looks great thanks.

I'll post comments inline and we'll get this merged soon

File: examples/contour.jl [hunk 1]
```diff
@@ -13,3 +36,184 @@ function contour1()
...
+                    xaxis2=attr(;domain=[0.55,1], anchor="y2"),
+                    yaxis2=attr(;domain=[0.55,1], anchor="x2"),
+                    xaxis3=attr(;domain=[0,0.45], anchor="y3"),
+                    yaxis3=attr(;domain=[0,0.45], anchor="x3"),
+                    xaxis4=attr(;domain=[0.55,1], anchor="y4"),
+                    yaxis4=attr(;domain=[0,0.45], anchor="x4"))
+    plot(data, layout)
+end
+
+function contour8()
```
Comment (spencerlyon2) [inline examples/contour.jl]: Similar comment for this example

```julia
function contour8()
    z = [2  4   7   12  13  14  15  16
         3  1   6   11  12  13  16  17
         4  2   7   7   11  14  17  18
         5  3   8   8   13  15  18  19
         7  4   10  9   16  18  20  19
         9  10  5   27  23  21  21  21
         11 14  17  26  25  24  23  22]
    
    p1 = plot(contour(;z=z, line_smoothing=0))
    p2 = plot(contour(;z=z, line_smoothing=0.85))

    p = [p1 p2]

    relayout!(p, title="Smoothing Contour Lines")

    p
end
```

Comment (caimichael) [inline examples/contour.jl]: I was wondering how you are able to exclude the domain/range information (x1,x2,y1,y2)/what exactly relayout! does?

Comment (spencerlyon2) [inline examples/contour.jl]: Check out [this section](http://spencerlyon.com/PlotlyJS.jl/manipulating_plots/#subplots) of the documentation for how I avoid setting the axis domain/range information.

`relayout!` simply adjusts fields of the plot's layout. 

Roughly what I do is 

1. create two separate plots, each containing one contour.
2. Use the notation introduced in the doc section I linked to to combine them into one plot with two subplots
3. Adjust the layout on this new plot to have the title we want

**Comment by spencerlyon2** [comment]
Looks great, thanks!

**Pull request merged**